.github/workflows/ci.yaml

```diff
@@ -142,6 +155,19 @@ jobs:
     - name: Ping frontend
       run: |
           curl --silent --output /dev/null --show-error --fail localhost:3000
-    - name: UI tests
+    - name: Run UI tests (Cypress)
       run: |
           docker run -t -v $PWD/.github/workflows/ui-tests:/e2e -w /e2e --network=host cypress/included:4.3.0
```

Comment:
If we used the external ip address instead this would get updated to include `-e CYPRESS_baseURL=${EXTERNAL_IP}`

Reply:
Thanks! updated.